### ISSUE TITLE
The extension does not respect the pythonpath VSCode setting

### ISSUE THREAD

**Issue by lengfengbyit** [iansan5653/vscode-format-python-docstrings#15]
Unknown Error: Could not format docstrings.

**Comment by iansan5653** [comment]
Hello @lengfengbyit! I'd like to fix your issue but I need some more information. If you can, can you post a sample of the text you are trying to format? Also if you could provide the version number of the extension that would be great. Finally, please try running the command `docformatter` on your command line and let me know what you see. 

It's possible this is related to a bug I resolved recently. Are there any spaces in the path to the file you are formatting?

If you could give me this information I will work to resolve the issue as quickly as possible.

**Comment by lengfengbyit** [comment]
The formatted content is a very simple python file, even if there is a line of code inside, such as: a = "hello worl" will also format failure, no space or special characters in the file path.
In addition, I don't know **docformatter** This command is executed, and this command is not supported in vscode.

**Comment by iansan5653** [comment]
Thanks for the followup! I think I may have an idea of what the problem is. If you could just do the following I can confirm this.

If you open a new terminal on your device and run the command `docformatter`, do you see something like the following?
> ```
> 'docformatter' is not recognized as an internal or external command,
> operable program or batch file.
> ```

What about if you run `pip`? 

Are you using virtual environments?

**Comment by iansan5653** [comment]
I thought the problem might be with virtual environments but it looks like that's not it. Please feel free to comment and reopen with more info.

**Comment by Setkias** [comment]
I experience the same error and another error:

Command failed: docformatter "c:\Users\b\Documents\code\std\app.py" --wrap-summaries 79 --wrap-descriptions 72 docformatter wordt niet herkend als een interne of externe opdracht, programma of batchbestand.

when I run docformatter through Ctrl-Shift-P.
When I run docformatter in the terminal it runs fine.

I used conda to setup my virtual environment and pip and docformatter are installed.

**Comment by iansan5653** [comment]
@Setkias Thanks for the information. If you open a new terminal elsewhere (not in the virtual environment or in the directory with the virtual environment) are you still able to run `docformatter` without issues?

**Comment by Setkias** [comment]
No. I have only one docformatter installation, and it lives inside the virtual environment. 
I have set the pythonpath in my workspace settings.json:

`    "python.pythonPath": "C:\\tools\\Anaconda3\\envs\\test\\python.exe",`

When I run:

`   c:\tools\Anaconda3\envs\test\python.exe -m docformatter`

in a shell where the environment is not activated I get no error.

When i run `python -m docformatter` from that same shell without the full path, using the python version that is the first found in my PATH variable, I do get a 'not found' error.

In short, yes, I can make it work by installing docformatter in another python that you call 'global' (which is a different version than the one that I use for running, debugging, linting, formatting, testing in vscode) but it feels counter intuitive. 

Would it not be correct to use the interpreter selected in vscode instead of a global python?

**Comment by iansan5653** [comment]
You're definitely right - this is a scenario I hadn't thought of. I just need to have it check the VSCode setting before defaulting to the Python in PATH. Thanks for the info! I'll work on it soon (unless you'd like to submit a PR).

**Comment by iansan5653** [comment]
@Setkias This should be resolved with the latest update. Let me know if you still run into any issues.
